python_ta/checkers/invalid_name_checker.py

```diff
@@ -291,20 +296,42 @@ class InvalidNameChecker(BaseChecker):
             "Used when the name doesn't conform to standard Python naming conventions.",
         ),
     }
+    options = (
+        (
+            "ignore-names",
+            {
+                "default": (),
+                "type": "csv",
```

Comment:
Instead use `regexp`; see, for example, https://github.com/pylint-dev/pylint/blob/6a09d298b2313ce143d986502b1d4d2dcd872b18/pylint/checkers/variables.py#L1230-L1237. I think you can set the default to be an empty string. This means the user should specify just a single regular expression rather than a list of regular expressions, but I think that makes sense.

Same for the other option.

Reply:
I changed this and made other related changes. I had to include`re.compile()` in the config for the test class because the string is not turned automatically into a `re.Pattern` object as it happens in the checker class directly.